doc/operator-notes.md

```diff
@@ -30,3 +30,35 @@ In the second two cases, where there is a preexisting filesystem, Ignition's beh
 If `wipeFilesystem` is set to true, Ignition will always wipe any preexisting filesystem and create the desired filesystem. Note this will result in any data on the old filesystem being lost.
 
 If `wipeFilesystem` is set to false, Ignition will then attempt to reuse the existing filesystem. If the filesystem is of the correct type, has a matching label, and has a matching UUID, then Ignition will reuse the filesystem. If the label or UUID is not set in the Ignition config, they don't need to match for Ignition to reuse the filesystem. Any preexisting data will be left on the device and will be available to the installation. If the preexisting filesystem is *not* of the correct type, then Ignition will fail, and the machine will fail to boot.
+
+## SELinux
+
+When using Ignition with distributions which rely on [SELinux][selinux] for additional security, extra care must be taken to prevent Ignition from breaking the system. Unfortunately, distributions do not typically include SELinux policies in the initramfs where Ignition runs, so any files, directories, and links created by Ignition don't receive the proper default SELinux labels.
+
+A workaround for this issue is to use [`restorecon`][restorecon] in a oneshot systemd unit to relabel files that Ignition has touched. This unit can be set to run after the SELinux policies have loaded, but before services will try to use them.
```

Comment:
Once you get the file system with the selinux policy is mounted somewhere you should be able to apply labels with something like `setfiles -F ${sysroot}/usr/lib/selinux/${policy}/contexts/files/file_contexts $[path_to_file}`.

Reply:
Apologies for my lack of selinux knowledge, but what would I fill in for `${policy}`? I don't even have a `/usr/lib/selinux` on my f27 laptop.

Reply:
That path is for Container Linux, maybe `/var/lib/selinux` or `/etc/selinux` will work for fedora.  There are usually multiple policies installed.  You would need to use the right policy files when you make any labels.  Container Linux only includes the 'mcs' policy, so `policy=msc`.  Sorry, I don't know about fedora.  Maybe something like `sudo find / -name '*contexts'` will give something.